File: src/main/java/com/hyungu/whoseturn/repositories/MemberRepository.java
package com.hyungu.whoseturn.repositories;

import com.hyungu.whoseturn.entities.Member;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.repository.CrudRepository;

import java.util.List;

public interface MemberRepository extends CrudRepository<Member, String> {

    List<Member> findAllMembers();

}

File: src/main/kotlin/com/hyungu/whoseturn/repositories/MemberRepository.kt
package com.hyungu.whoseturn.repositories

import com.hyungu.whoseturn.entities.MemberEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.repository.CrudRepository

interface MemberRepository : JpaRepository<MemberEntity, Long>
interface MemberCrudRepository : CrudRepository<MemberEntity, Long>
